chapter1/index.js

```diff
@@ -1,33 +1,87 @@
-(function(){
+let Formulas = {};
 
-  console.log('Hello world');
+Formulas.add = function(a, b){
+  return a + b;
+}
 
-  let add = function(a, b){
-    return a + b;
+Formulas.divideMultiply = function(a, b){
+  if(b==0){
+    return undefined;
   }
+  return a * a / b;
+}
 
-  let o = {
-     key: { key: { key: 42 }}
-  };
+Formulas.roundNumbers = function(a){
+  return Math.round(a);
+}
 
-  let arr = [1,2,3,4,5];
-  
-  console.log(arr[4])
+Formulas.sinTimesCos = function(a){
+  return Math.sin(a)*Math.cos(a);
+}
 
+Formulas.parseNumberFromString = function(a){
+  return parseFloat(a);
+}
 
+Formulas.findObjectByKey = function(obj, key){
+  var i;
+  var j;
+  var p;
+  for(i = 0; i < Object.keys(obj).length; i++){
```

Comment:
you could use `let` instead of `var` here. It behaves more like *normal* variable from Java.